build-tools/build-indexes

```diff
@@ -9,7 +9,7 @@ const rootDir = path.resolve(__dirname, '..');
 process.chdir(rootDir);
 
 if (!fs.existsSync('data/pokemon-showdown')) {
-	child_process.execSync('git clone https://github.com/smogon/pokemon-showdown.git', {
+	child_process.execSync('git clone https://github.com/scoopapa/DH.git pokemon-showdown', {
```

Comment:
```suggestion
	child_process.execSync('git clone https://github.com/smogon/pokemon-showdown.git pokemon-showdown', {
```

Reply:
will do, was only keeping it in case someone wanted to download the files to verify that it works